File: android/src/main/java/com/bishare/protocol/constants/Ports.kt
package com.bishare.protocol.constants

/**
 * BIShare network port definitions.
 * All ports are in the dynamic/private range (49152–65535).
 */
object BISharePort {
    /** Main transfer server — TCP HTTP */
    const val MAIN: Int = 58317

    /** QUIC transport + Clipboard sync — UDP */
    const val QUIC: Int = 58318

    /** Transfer Rooms — TCP HTTP */
    const val ROOM: Int = 58319

    /** WebDAV Server — TCP */
    const val WEBDAV: Int = 58320
}

File: ios/Sources/BIShareProtocol/Constants/Ports.swift
import Foundation

/// BIShare network port definitions.
/// All ports are in the dynamic/private range (49152–65535).
public enum BISharePort {
    /// Main transfer server — TCP HTTP (was 53317, now independent from LocalSend)
    public static let main: UInt16 = 58317

    /// QUIC transport + Clipboard sync — UDP (was 53318)
    public static let quic: UInt16 = 58318

    /// Transfer Rooms — TCP HTTP (was 53319)
    public static let room: UInt16 = 58319

    /// WebDAV Server — TCP (was 53320, iOS/macOS only)
    public static let webdav: UInt16 = 58320
}
